slack-api-client/src/main/java/com/slack/api/methods/MethodsRateLimits.java

```diff
@@ -435,7 +435,7 @@ public static void setRateLimitTier(String methodName, MethodsRateLimitTier tier
         setRateLimitTier(USERS_SET_PHOTO, Tier2);
         setRateLimitTier(USERS_SET_PRESENCE, Tier2);
         setRateLimitTier(USERS_PROFILE_GET, Tier4);
-        setRateLimitTier(USERS_PROFILE_SET, Tier4);
+        setRateLimitTier(USERS_PROFILE_SET, Tier2);
```

Comment:
The difference between documents could be confusing but the rate limit is Tier 3 on the API endpoint side: https://api.slack.com/methods/users.profile.set Thus, we'd like to use the setting (Tier 3) for it.

Reply:
Sounds good, will do.